capitalize_decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'name_decorator'
+
+class CapitalizeDecorator < NameDecorator
```

Comment:
- [ ] Your code produces an error, kindly make sure to fix this so your Decorator is working as expected

![Screenshot_20230308_214214](https://user-images.githubusercontent.com/25227022/223844810-0a4fb004-2a2c-4f6b-982b-a890e63e9715.png)